content/_index/users/index.md

```diff
@@ -0,0 +1,9 @@
++++
+date = "2018-07-09"
+fragment = "portfolio"
+weight = 300
+background = "light"
+
+title = "Users"
+subtitle = "A variety of using Kubernetes Event-driven Autoscaling (KEDA) users are:"
```

Comment:
I'd rephrase the subtitle a little bit, "using", "users".

Reply:
Something like this then?

`A variety of users are autoscaling applications with Kubernetes Event-driven Autoscaling (KEDA):`

Reply:
Yeah, sounds good!